### PR TITLE
🌱 Release v1.5.1

### PR DESCRIPTION
**Pull request by g-gaston**
/area release
/hold

### CODE REVIEW

File: CHANGELOG/v1.5.1.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+## Changes since v1.5.0
+---
+## :chart_with_upwards_trend: Overview
+- 33 new commits merged
+- 2 feature additions ✨
+- 9 bugs fixed 🐛
+
+## :sparkles: New Features
+- MachineSet: Machineset: adjust preflight check to allow kubelet version skew of 3 for clusters running v1.28 and above (#9233)
```
Comment (furkatgofurov7) [inline CHANGELOG/v1.5.1.md:16]: can we please fix the duplication here?

File: CHANGELOG/v1.5.1.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- Testing/documentation: V1.28: Prepare quickstart, capd and tests for the new release includi… (#9225)
+
+## :bug: Bug Fixes
+- bootstrap: Certificate paths in cloud-init scripts should not use a platform-dependent path separator (#9230)
+- CAPD: CAPD: delete container after failed start to work around port allocation issues (#9130)
```
Comment (furkatgofurov7) [inline CHANGELOG/v1.5.1.md:21]: same

File: CHANGELOG/v1.5.1.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- ClusterClass: Add topology-owned label to MachineHealthChecks. (#9204)
+- ClusterClass: Fix ClusterClass enqueue for ExtensionConfig (#9137)
+- Dependency: Bump golang.org/x/net to v0.13.0 (#9122)
+- Devtools: Fix tilt-prepare leader-elect setting (#9316)
+- e2e: Test: pin conformance image to a version which includes a fix for the dualstack tests (#9256)
+- KCP: Allow unsetting useExperimentalRetryJoin field from KubeadmControlPlane (#9202)
+- util: Fix AddAnnotations for unstructured.Unstructured (#9176)
+
+## :seedling: Others
+- CAPD: CAPD: fix multi error handling in RunContainer (#9243)
```
Comment (furkatgofurov7) [inline CHANGELOG/v1.5.1.md:31]: same

File: CHANGELOG/v1.5.1.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- Testing/documentation: V1.28: Prepare quickstart, capd and tests for the new release includi… (#9225)
+
+## :bug: Bug Fixes
+- bootstrap: Certificate paths in cloud-init scripts should not use a platform-dependent path separator (#9230)
+- CAPD: CAPD: delete container after failed start to work around port allocation issues (#9130)
+- ClusterClass: Add topology-owned label to MachineHealthChecks. (#9204)
+- ClusterClass: Fix ClusterClass enqueue for ExtensionConfig (#9137)
+- Dependency: Bump golang.org/x/net to v0.13.0 (#9122)
+- Devtools: Fix tilt-prepare leader-elect setting (#9316)
+- e2e: Test: pin conformance image to a version which includes a fix for the dualstack tests (#9256)
```
Comment (furkatgofurov7) [inline CHANGELOG/v1.5.1.md:26]: same

**Review by furkatgofurov7 — COMMENTED** [review]
Thanks for the PR. 

Few nits:

**Comment by furkatgofurov7** [comment]
cc @kubernetes-sigs/cluster-api-release-team
(FYI: @kubernetes-sigs/cluster-api-maintainers)

File: CHANGELOG/v1.5.1.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+## Changes since v1.5.0
+---
+## :chart_with_upwards_trend: Overview
+- 33 new commits merged
+- 2 feature additions ✨
+- 9 bugs fixed 🐛
+
+## :sparkles: New Features
+- MachineSet: Adjust preflight check to allow kubelet version skew of 3 for clusters running v1.28 and above (#9233)
+- Testing/documentation: V1.28: Prepare quickstart, capd and tests for the new release includi… (#9225)
```
Comment (sbueringer) [inline CHANGELOG/v1.5.1.md:17]:
```suggestion
- Testing/documentation: v1.28: Prepare quickstart, capd and tests for the new release including kind bump (#9225)
```
nit

File: CHANGELOG/v1.5.1.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+## :bug: Bug Fixes
+- bootstrap: Certificate paths in cloud-init scripts should not use a platform-dependent path separator (#9230)
+- CAPD: Delete container after failed start to work around port allocation issues (#9130)
+- ClusterClass: Add topology-owned label to MachineHealthChecks. (#9204)
+- ClusterClass: Fix ClusterClass enqueue for ExtensionConfig (#9137)
+- Dependency: Bump golang.org/x/net to v0.13.0 (#9122)
```
Comment (sbueringer) [inline CHANGELOG/v1.5.1.md:24]: Might want to move this down here as well

**Review by sbueringer — COMMENTED** [review]
overall lgtm, just some nits

File: CHANGELOG/v1.5.1.md (hunk 5 above)
Comment (furkatgofurov7) [reply CHANGELOG/v1.5.1.md:17]: hmm, good catch that is due to long PR title + cherry pick PR prefix added making this

**Review by vincepri — APPROVED** [review]
/approve

File: CHANGELOG/v1.5.1.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+- 33 new commits merged
+- 2 feature additions ✨
+- 9 bugs fixed 🐛
+
+## :sparkles: New Features
+- MachineSet: Adjust preflight check to allow kubelet version skew of 3 for clusters running v1.28 and above (#9233)
+- Testing/documentation: v1.28: Prepare quickstart, capd and tests for the new release including kind bump (#9225)
+
+## :bug: Bug Fixes
+- bootstrap: Certificate paths in cloud-init scripts should not use a platform-dependent path separator (#9230)
```
Comment (killianmuldoon) [inline CHANGELOG/v1.5.1.md:20]: This should be CAPBK - my bad in originally labelling the PR wrong.

**Review by killianmuldoon — COMMENTED** [review]
Just a nit - love the new way of adding / reviewing release notes!

**Review by CecileRobertMichon — COMMENTED** [review]
/lgtm

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

**Pull request merged**